src/test/java/com/android/volley/AsyncRequestQueueTest.java

```diff
@@ -161,4 +143,57 @@ public void removeRequestEventListener_removesListener() throws Exception {
         verifyNoMoreInteractions(listener);
         queue.stop();
     }
+
+    @Test
+    public void requestsQueuedBeforeCacheInitialization_asyncCache() {
+        AsyncCache mockAsyncCache = mock(AsyncCache.class);
+        queue = createRequestQueue(mockAsyncCache);
```

Comment:
Could `queue` be a local var instead? Since it's not the one set during setup?

Reply:
Sure, done